dhcp/unikernel.ml

```diff
@@ -0,0 +1,85 @@
+open V1_LWT
+open Lwt
+
+(* IP Configuration, all you need besides dhcpd.conf. *)
+let ipaddr = Ipaddr.V4.of_string_exn "192.168.1.5"
```

Comment:
it'd be very easy to do with the new functoria, see https://github.com/Drup/mirage-skeleton/blob/functoria/hello/config.ml#L3-L5